### PR TITLE
Split out max_density test and add owners to other perf tests

### PR DESCRIPTION
**Pull request by gmarek**
@rrati - IIRC you're the one who added max-density test and promised to look after it. I split it out of the main density.go, so it'd be clear that this one is not owned by us.

cc @kubernetes/sig-scalability-misc

### CODE REVIEW

File: test/e2e/perf/max_density.go [hunk 1]
```diff
@@ -0,0 +1,167 @@
+/*
```
Comment (wojtek-t) [inline test/e2e/perf/max_density.go]: Personally - I vote for removing this test.

**Comment by gmarek** [comment]
Ping @rrati - I'm going to remove that test on Friday if no one is willing to claim it.

**Comment by rrati** [comment]
@gmarek I'm ok removing it.  I'm not sure it was ever used as was intended.

**Pull request merged**